PWGHF/D2H/TableProducer/candidateCreatorCharmResoReduced.cxx

```diff
@@ -130,7 +135,7 @@ struct HfCandidateCreatorCharmResoReduced {
     if (channel == DecayChannel::Ds2StarToDplusK0s || channel == DecayChannel::XcToDplusLambda || channel == DecayChannel::LambdaDminus) {
       massD = massDplus;
     } else if (channel == DecayChannel::Ds1ToDstarK0s) {
-      massD = massDstar;
+      massD = massDstar - massD0;
```

Comment:
This is not correct because we compute the invariant mass of D*+ and K0S, having the deltaM in the formula would not make sense. To compute the invariant mass different as in Run2 (i.e. M(KpipiK0S) - M(Kpipi)) one should use all the daughters, but we do not store them. Since we verified that we get more or less the same resolution simply by using the PDG mass also for the D in the decay, it was correct before:
```suggestion
      massD = massDstar;
```

Reply:
Hi Fabrizio, I agree with you. However in the context of the function `isDSelected`, i am initializing the mass of the D* as a difference in order to apply a cut on the invariant mass of the Dstar daughter only, and currently I am saving the mass difference for the D*. In the computation of the resonance invariant mass (line 221) i am using the PDG masses

Reply:
Ah good! Sorry, I misunderstood it. If you are anyway computing the invariant mass using the D* mass and not the delta it's fine. Thanks for the explanation!